### ISSUE TITLE
Research startup options available in other protocols

### ISSUE THREAD

**Issue by scasplte2** [Topl/Bifrost#1035]
## Summary
In order to increase accessibility for users running nodes, a robust suite of command-line tools and options should be provided. There are many parameters and user-defined variables that can be placed in the configuration files, and many of these are useful at the CLI level when testing or running many nodes programmatically. A detailed overview of current node and wallet options to add is a good start to prioritizing CLI options to be added to both the node and Gjallarhorn.

## Acceptance criteria
- a document summarizing useful CLI options and concepts

**Comment by Tuxman** [comment]
A document has been created in Confluence.
